qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -3261,7 +3259,16 @@ void FitPropertyBrowser::addAllowedSpectra(const QString &wsName, const QList<in
     for (auto const i : wsSpectra) {
       indices.push_back(static_cast<int>(ws->getIndexFromSpectrumNumber(i)));
     }
+    auto wsFound = m_allowedSpectra.find(wsName);
     m_allowedSpectra.insert(wsName, indices);
+    if (wsFound != m_allowedSpectra.end()) {
```

Comment:
This doesn't need to be done now, but just a heads-up for the future we can write this all in the if block

```c++
if (auto wsFound = m_allowedSpectra.find(wsName); wsFound!=m_allowedSpectra.end()) {